requirements.txt

```diff
@@ -1,2 +1,42 @@
-aiohttp>=3.6,<4
-protobuf>=3.0,<4
\ No newline at end of file
+aiofiles==0.6.0
+aiohttp==3.7.3
+appdirs==1.4.4
+appnope==0.1.2
+argcomplete==1.12.2
+async-timeout==3.0.1
+attrs==20.3.0
+backcall==0.2.0
+chardet==3.0.4
+colorlog==4.7.2
+decorator==4.4.2
+distlib==0.3.1
+filelock==3.0.12
+freezegun==1.1.0
+idna==3.1
+iniconfig==1.1.1
+ipython==7.20.0
+ipython-genutils==0.2.0
+jedi==0.18.0
+later==20.10.1
+multidict==5.1.0
+nox==2020.12.31
+packaging==20.9
+parso==0.8.1
+pexpect==4.8.0
+pickleshare==0.7.5
+pluggy==0.13.1
+prompt-toolkit==3.0.16
+protobuf==3.15.1
+ptyprocess==0.7.0
+py==1.10.0
+Pygments==2.8.0
+pyparsing==2.4.7
+pytest==6.2.2
+python-dateutil==2.8.1
+six==1.15.0
+toml==0.10.2
+traitlets==5.0.5
+typing-extensions==3.7.4.3
+virtualenv==20.4.2
+wcwidth==0.2.5
+yarl==1.6.3
```

Comment:
I still really do not like like. Sorry. Please revert it and add aiofiles. If desired, make a requirements dev version, but will pick requirements manually.

Reply:
Fine, we keep it frozen. It really looks ugly and you have no idea, which packages are actually required because of the dependency chain